woocommerce/woocommerce-android
# Show products information in analytics hub

Closes: #5237

### Description
This pull request should close the analytics hub part 1. Here the new list card #5442 would be used to display the last card of the Hub.

### Testing instructions
There are unit tests for the information and to manual test please, follow this steps:

1. Open the application compiled in debug mode
2. Click on analytics section icon on bottom navigation bar
3. Scroll down to see the products card
4. Products card should appear at bottom

### Images/gif

<img src="https://user-images.githubusercontent.com/572601/145792113-852b5638-94a0-4fda-a01a-96832e49ba42.png" width="400" height="1000">
<img src="https://user-images.githubusercontent.com/572601/145792120-5c58196c-36aa-40d0-9783-d2de720196d4.png" width="400" height="1000">
<img src="https://user-images.githubusercontent.com/572601/145792122-bedcb8d6-523e-4a49-b310-1a5ab6496508.png" width="400" height="1000">
<img src="https://user-images.githubusercontent.com/572601/145792125-608c0e0c-1d57-4a78-ab3f-8255add4b715.png" width="400" height="1000">
<img src="https://user-images.githubusercontent.com/572601/145792128-04bb8210-13c6-4f4e-9586-0315c673d436.png" width="400" height="1000">
<img src="https://user-images.githubusercontent.com/572601/145792136-b4f6dddc-e3e1-4f19-a413-0c2855662561.png" width="400" height="1000">
<img src="https://user-images.githubusercontent.com/572601/145792640-e4a0c91b-5964-4438-93b9-7ff86bcdee77.png" width="400" height="1000">
